Assets/Scripts/Managers/DayCycleManager.cs

```diff
@@ -84,6 +84,12 @@ void MorningReset()
 		Invoke( "StartMidnightOverlay", timeUntilBlackout );
 	}
 
+	public float GetNormalizeCurrentTime()
```

Comment:
This should be `GetNormalizedCurrentTime()` (you forgot the 'd' in "Normalized"). Also, you should add a comment noting that "normaized" in this case means that it factors out time before the start of the day.